backends/gpu/lib/wrapper/miopen_wrapper.cc

```diff
@@ -361,6 +361,17 @@ llvm::Error MiopenPoolingForward(
   return TO_ERROR(miopenPoolingForward(
       handle, pooling_desc, ToRocm(alpha), x_desc, ToRocm(x), ToRocm(beta),
       y_desc, ToRocm(y), do_backward, ToRocm(workspace), workspace_size_bytes));
+} 
+
+llvm::Error  MiopenPoolingGetWorkSpaceSizeV2(
```

Comment:
Please change signature to `llvm::Expected<size_t> MiopenPoolingGetWorkSpaceSize(...)`.